src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -22,7 +22,14 @@ public static class CredentialCacheTest
 
         private static readonly string authenticationType1 = "authenticationType1";
         private static readonly string authenticationType2 = "authenticationType2";
+        
+        private static readonly string authenticationTypeNTLM = "NTLM";
+        private static readonly string authenticationTypeKerberos = "Kerberos";
+        private static readonly string authenticationTypeNegotiate = "Negotiate";
+        private static readonly string authenticationTypeBasic = "Basic";
+        private static readonly string authenticationTypeDigest = "Digest";
 
+        private static readonly NetworkCredential customCredential = new NetworkCredential("username", "password");
         private static readonly NetworkCredential credential1 = new NetworkCredential("username1", "password");
         private static readonly NetworkCredential credential2 = new NetworkCredential("username2", "password");
         private static readonly NetworkCredential credential3 = new NetworkCredential("username3", "password");
```

Comment:
>Because I don't want to make modification to old test code.

Usually we try not to touch areas of tests or code that are unaffected by the change.  But you have done a lot of changes already and it just makes sense to consolidate and clean this up.  So, yes, do that.